src/Components/MainApp/App.js

```diff
@@ -1,50 +1,48 @@
-import '../../CSS/MainApp/App.css';
-import React, { useState } from 'react';
-import Header from './MainAppHeader.js';
-import ExperimentOne from '../ExperimentOne/ExperimentOne.js';
-import ExperimentTwo from '../ExperimentTwo/ExperimentTwo.js';
-
+import "../../CSS/MainApp/App.css";
+import React, { useState } from "react";
+import Header from "./MainAppHeader.js";
+import ExperimentOne from "../ExperimentOne/ExperimentOne.js";
+import ExperimentTwo from "../ExperimentTwo/ExperimentTwo.js";
 
 function App() {
   // Button States
-  const [headerState, setHeaderState]=useState(true);
-  const [experimentOneButtonState,setExperimentOneButtonState ]=useState(false);
-  const [experimentTwoButtonState,setExperimentTwoButtonState]=useState(false);
-
-  const handleExperimentOneEvent=()=>
-  {
-    setExperimentOneButtonState(true);
-    setExperimentTwoButtonState(false);
-    setHeaderState(false);
-  }
-
-  const handleExperimentTwoEvent=()=>
-  {
-    setExperimentTwoButtonState(true);
-    setExperimentOneButtonState(false);
-    setHeaderState(false);
-  }
-
-  const backButtonState=()=>
-  {
-    setExperimentOneButtonState(false);
-    setExperimentTwoButtonState(false);
-    setHeaderState(true);
-  }
+  const [experimentButtonState, setExperimentButtonState] = useState("0");
```

Comment:
Would probably be better with a number here too... also maybe the default value should be `null` rather than `0` just because I guess it's possible for an experiment id to be 0

Reply:
Hmm. Yeah I agree that the id can be zero. I agree and so will modify this to null